src/App.tsx

```diff
@@ -1,7 +1,13 @@
-import React from 'react';
+import { Component } from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
+type State = {
+  selectedGood: string;
+  title: string;
```

Comment:
title variable in state is redundant, delete it

Reply:
done